lib/facade_converter.ts

```diff
@@ -95,6 +95,7 @@ export class FacadeConverter extends base.TranspilerBase {
   emitExtraImports(sourceFile: ts.SourceFile) {
     var libraries = <ts.Map<string>>{
       "XMLHttpRequest": "dart:html",
+      "KeyboardEvent": "dart:html",
```

Comment:
Nope, I tried that but we cast them to dynamic and then they are erased,
and we get an error from dartanalyze because the dart:html import goes
unused.

On Fri, Oct 9, 2015 at 11:22 AM Martin Probst <notifications@github.com>
wrote:

> In lib/facade_converter.ts
> <https://github.com/angular/ts2dart/pull/300#discussion_r41662136>:
>
> > @@ -95,6 +95,7 @@ export class FacadeConverter extends base.TranspilerBase {
> >    emitExtraImports(sourceFile: ts.SourceFile) {
> >      var libraries = <ts.Map<string>>{
> >        "XMLHttpRequest": "dart:html",
> > +      "KeyboardEvent": "dart:html",
>
> What about MouseEvent? Could we add some more of the usual suspects here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular/ts2dart/pull/300/files#r41662136>.
>